_partials/_migrate_live_migration_step2.md

```diff
@@ -2,6 +2,12 @@ import SetupSourceTarget from "versionContent/_partials/_migrate_set_up_source_a
 
 <SetupSourceTarget />
 
+<Highlight type="important">
+When specifying a `$TARGET` URI for live migration, use the direct PostgreSQL URI
+instead of a connection pool from Timescale instance. Connection pools may
+close connections due to inactivity, which can cause problems during live migration.
```

Comment:
Two things:
1. I reformulated the wording a little
2. There are other failure modes of connection pooling

```suggestion
Do not use a Timescale connection pooler connection for live migration. There
are a number of issues which can arise when using a connection pooler, and no
advantage. Very small instances may not have enough connections configured by
default, in which case you should modify the value of `max_connections`, in
your instance, as shown on [Configure database parameters](/use-timescale/:currentVersion/configuration/customize-configuration/#configure-database-parameters).
```